internal/libdocker/builder.go

```diff
@@ -73,13 +73,16 @@ func (b *Builder) BuildSimulatorImage(ctx context.Context, name string) (string,
 	dir := b.config.Inventory.SimulatorDirectory(name)
 	buildContextPath := dir
 	buildDockerfile := "Dockerfile"
+	if len(b.config.OverrideDockerfile) > 0 {
```

Comment:
Comparison with string `nil` value looks more appropriate from my view:
```suggestion
	if b.config.OverrideDockerfile != "" {
```
@stdevMac what do you think?

Reply:
Yeah, it's more Go idiomatic, good catch